ext/mixed/js/core.js

```diff
@@ -250,15 +250,35 @@ class EventListenerCollection {
         return this._eventListeners.length;
     }
 
-    addEventListener(node, type, listener, options) {
-        node.addEventListener(type, listener, options);
-        this._eventListeners.push([node, type, listener, options]);
+    addEventListener(object, ...args) {
+        object.addEventListener(...args);
+        this._eventListeners.push(['removeEventListener', object, ...args]);
+    }
+
+    addListener(object, ...args) {
+        object.addListener(args);
+        this._eventListeners.push(['addListener', object, args]);
```

Comment:
Others use the remove function name, and the switch case below is for `removeListener`.

Reply:
a8b39b7e1726c288f52fdc7d9255803de33ce7e3